docs/UserGuide.adoc

```diff
@@ -32,6 +35,29 @@ This user guide provides a quick start guide for you to easily setup and install
 
 {zwsp}
 
+=== Terminology
+For ease of communication, this document will refer to lessons/activities/events/appointments that you might add into the planner as _slots_.
+
+Additionally, throughout this user guide, there will be various icons used as described below.
+
+[TIP]
+This is a tip. Follow these suggested tips to make your life much simpler when using _PlanMySem_!
+
+[NOTE]
+This is a note. These are things for you to take note of when using _PlanMySem_.
+
+[IMPORTANT]
+This is a sign-post dictating important information. These are information that you will surely need to know to use _PlanMySem_ properly.
+
+[CAUTION]
+This is a sign-post informing caution. Please take note of these items and exercise some care.
+
+[WARNING]
+This is a rule. Ensure that you follow these rule to make your life using _PlanMySem_ a pleasant one.
+{zwsp}
+
```

Comment:
I don't think we need all of these icons here and we should only put those we use. I think this section can be under the "About this user guide" section which i added in my PR.